app/renderer/components/navigation/urlBar.js

```diff
@@ -489,7 +493,7 @@ class UrlBar extends React.Component {
   }
 
   get showEvCert () {
-    if (this.props.titleMode) {
+    if (this.props.titleMode || this.props.isActive) {
```

Comment:
awesome, this simplifies things a lot :)

Reply:
Can you check my squash and rebase?